public/404.html

```diff
@@ -1,11812 +1,278 @@
-        }
-        .navbar.is-primary .navbar-start > a.navbar-item:focus,
-        .navbar.is-primary .navbar-start > a.navbar-item:hover,
-        .navbar.is-primary .navbar-start > a.navbar-item.is-active,
-        .navbar.is-primary .navbar-start .navbar-link:focus,
-        .navbar.is-primary .navbar-start .navbar-link:hover,
-        .navbar.is-primary .navbar-start .navbar-link.is-active,
-        .navbar.is-primary .navbar-end > a.navbar-item:focus,
-        .navbar.is-primary .navbar-end > a.navbar-item:hover,
-        .navbar.is-primary .navbar-end > a.navbar-item.is-active,
-        .navbar.is-primary .navbar-end .navbar-link:focus,
-        .navbar.is-primary .navbar-end .navbar-link:hover,
-        .navbar.is-primary .navbar-end .navbar-link.is-active {
-          background-color: #16213b;
-          color: #fff;
-        }
-        .navbar.is-primary .navbar-start .navbar-link::after,
-        .navbar.is-primary .navbar-end .navbar-link::after {
-          border-color: #fff;
-        }
-        .navbar.is-primary .navbar-item.has-dropdown:focus .navbar-link,
-        .navbar.is-primary .navbar-item.has-dropdown:hover .navbar-link,
-        .navbar.is-primary .navbar-item.has-dropdown.is-active .navbar-link {
-          background-color: #16213b;
-          color: #fff;
-        }
-        .navbar.is-primary .navbar-dropdown a.navbar-item.is-active {
-          background-color: #1d2c4e;
-          color: #fff;
-        }
-      }
-      .navbar.is-link {
-        background-color: #92eaff;
-        color: #fff;
-      }
-      .navbar.is-link .navbar-brand > .navbar-item,
-      .navbar.is-link .navbar-brand .navbar-link {
-        color: #fff;
-      }
-      .navbar.is-link .navbar-brand > a.navbar-item:focus,
-      .navbar.is-link .navbar-brand > a.navbar-item:hover,
-      .navbar.is-link .navbar-brand > a.navbar-item.is-active,
-      .navbar.is-link .navbar-brand .navbar-link:focus,
-      .navbar.is-link .navbar-brand .navbar-link:hover,
-      .navbar.is-link .navbar-brand .navbar-link.is-active {
-        background-color: #79e5ff;
-        color: #fff;
-      }
-      .navbar.is-link .navbar-brand .navbar-link::after {
-        border-color: #fff;
-      }
-      .navbar.is-link .navbar-burger {
-        color: #fff;
-      }
-      @media screen and (min-width: 1024px) {
-        .navbar.is-link .navbar-start > .navbar-item,
-        .navbar.is-link .navbar-start .navbar-link,
-        .navbar.is-link .navbar-end > .navbar-item,
-        .navbar.is-link .navbar-end .navbar-link {
-          color: #fff;
-        }
-        .navbar.is-link .navbar-start > a.navbar-item:focus,
-        .navbar.is-link .navbar-start > a.navbar-item:hover,
-        .navbar.is-link .navbar-start > a.navbar-item.is-active,
-        .navbar.is-link .navbar-start .navbar-link:focus,
-        .navbar.is-link .navbar-start .navbar-link:hover,
-        .navbar.is-link .navbar-start .navbar-link.is-active,
-        .navbar.is-link .navba
```

Comment:
Do we are about linking out to events as well?

Reply:
Ah yes, I meant to build the dropdown but forgot